rmw_fastrtps_cpp/src/type_support_common.cpp

```diff
@@ -132,6 +147,56 @@ bool TypeSupport::deserializeROSmessage(
   return true;
 }
 
+bool TypeSupport::get_key_hash_from_ros_message(
+  void * ros_message,
+  eprosima::fastdds::rtps::InstanceHandle_t * ihandle,
+  bool force_md5,
+  const void * impl) const
```

Comment:
not blocking PR, this can be more modern and clearer approach.

```suggestion
  const void * [[maybe_unused]] impl) const
```

Reply:
06b640d